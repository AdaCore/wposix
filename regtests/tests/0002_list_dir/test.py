from test_support import *

gprbuild('demo2')
run('demo2')
