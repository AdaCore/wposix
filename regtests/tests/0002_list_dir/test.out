List directory dir
.
..
file.txt
README
toto
List directory .
.
..
file.txt
README
toto
List directory /tmp-does-not-exists
POSIX.POSIX_ERROR - For_Every_Directory_Entry : Error_Code =  2 : NO_SUCH_FILE_OR_DIRECTORY
List directory /tmp/toto-does-not-exists
POSIX.POSIX_ERROR - For_Every_Directory_Entry : Error_Code =  2 : NO_SUCH_FILE_OR_DIRECTORY
